recipes/conversation_watsonxassistant/README.md

```diff
@@ -15,60 +15,64 @@ This recipe requires a TJBot with a microphone, a speaker, a Neopixel LED, and o
+### Create instances of IBM Cloud AI services
+Create instances of the [watsonx Assistant](https://cloud.ibm.com/catalog/services/watsonx-assistant), [Speech to Text](https://cloud.ibm.com/catalog/services/speech-to-text), and [Text to Speech](https://cloud.ibm.com/catalog/services/text-to-speech) services. Download the authentication credentials file for each service. Combine each of these files into a single file named `ibm-credentials.env` and place it in the `tjbot/recipes/translator` folder. See `ibm-credentials.sample.env` for an example.
 
-Make a copy the default configuration file and update it with your service credentials and the assistant environment ID.
+Next, make a copy of TJBot's sample configuration file.
 
-    $ cp config.default.js config.js
-    $ nano config.js
-    <enter your credentials and the Assistant environment ID in the specified places>
+```sh
+$ cp tjbot.sample.toml tjbot.toml
+$ nano tjbot.toml
+```
 
-Set up watsonx Assistant using the following steps:
+Set up Watson Assistant using the following steps:
 
-1. Launch the watsonx Assistant tool and create a new assistant.
-2. From the left menu click "Actions."
-3. Click the gear icon at the top right.
-4. Select the "Upload/Download" tab.
-5. Upload the `tjbot-action-sample.json` file.
-6. From the left menu select "Assistant settings."
-7. Under "Assistant IDs and API details" click the "view details" button.
-8. Copy the "Draft Environment ID" and paste it into your `config.js` file.
+1. Launch the Watson Assistant tool and create a new assistant.
+2. Click "Add dialog skill" and then "Import Skill."
+3. Upload the `tjbot-action-sample.json` file.
+4. Go back to the Assistants screen and click the menu (with the three dots), and click "Settings."
+5. Click "API Details" in the left sidebar.
+6. Copy the "Assistant ID".
+
+In the `[Recipe]` section of your `tjbot.toml` file, fill in the `environmentId` configuration parameter with the Assistant ID you just retrieved.
 
 Run!
 
-    $ sudo node conversation.js
+    $ sudo node index.mjs
 
 > Note the `sudo` command. Root user access is required to run TJBot recipes.
 
-watsonx Assistant uses actions to guide the conversation. For example when you ask TJBot "Please introduce yourself", the action taken is to make an introduction. You can add your own new actions, but for now, we have started you off with a few actions:
+Watson conversation uses actions to route the flow of a conversation. For example when you ask TJBot "Please introduce yourself", the action is to make an introduction. You can add your own new actions, but for now, we have started you off with a few actions:
```

Comment:
Shouldn't this say "watsonx Assistant" since that's the name of the product?

Reply:
Yep!